app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<h2>Sign up</h2>
```

Comment:
This is probably  because you cannot use my email account in your devise. 

To run this appplication you need an email account properly configured. You need the 2-step verification for your gmail account for example as is done here:

https://help.warmupinbox.com/en/articles/4934806-configure-for-google-workplace-with-two-factor-authentication-2fa


and also and App password, you can generete it in the following link

https://myaccount.google.com/apppasswords?utm_source=google-account&utm_medium=myaccountsecurity&utm_campaign=tsv-settings&rapt=AEjHL4PWzrboEKqAJ1-uFiT_jc_JM7D04bZJ4RO6XgHr0kfA3EVo-zMEW8QSMS-wHxDfRHi83r9D_pwiNb9lH5DyMko0upx3EA


Then you can change the password and email in the files  ./config/environments/development.rb 
./config/environment.rb 
with your account information:

    user_name:      'edahigure@gmail.com',
    password:       'iewencoweo3892',
    domain:         'localhost:3000',
    address:       'smtp.gmail.com',
    port:          '587',
    authentication:  "plain",
    enable_starttls_auto: true

and also line 29 of file ./config/environments/initializers/devise.rb 
by
config.mailer_sender = 'edahigure@gmail.com'